src/mygrad/random/random/random.py

```diff
@@ -0,0 +1,25 @@
+from mygrad import Tensor
+import numpy as np
+
+
+def random(shape=None, constant=False):
+    """ Return random floats in the half-open interval [0.0, 1.0). 
+    Alias for random_sample to ease forward-porting to the new random API.
+    Parameters
+    
+    To create a random sample of a given shape on the interval [a, b), call 
+    (b-a) * random(shape) + a
+
+    
+    ----------
+    shape: int or tuple of ints, optional
+        Output shape. If the given shape is, e.g., (m, n, k), then m * n * k samples are drawn. Default is None, in which case a single value is returned.
```

Comment:
line length is too long

```suggestion
        Output shape. If the given shape is, e.g., (m, n, k), then m * n * k samples are drawn. 
        Default is None, in which case a single value is returned.
```